src/cpu/pred/bpred_unit.hh

```diff
@@ -407,6 +400,9 @@ class BPredUnit : public SimObject
      */
     void updateBTB(ThreadID tid, PredictorHistory *&bpu_history);
 
+  private:
+    typedef std::deque<PredictorHistory *> History;
```

Comment:
I believe this should be lowercase `history`. See [Naming](https://www.gem5.org/documentation/general_docs/development/coding_style/#naming)

Reply:
Really? Also for type definitions? Its not a variable declaration.